tests/unit/test_providers.py

```diff
@@ -0,0 +1,85 @@
+"""Tests generic for parser."""
+import json
+import os
+from pathlib import Path
+
+import pytest
+
+# pylint: disable=duplicate-code
+from circuit_maintenance_parser.providers import (
+    GenericProvider,
+    EUNetworks,
+    Lumen,
+    Megaport,
+    NTT,
+    PacketFabric,
+    Telstra,
+    Zayo,
+)
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+
+@pytest.mark.parametrize(
+    "provider_class, raw_file, results_file",
+    [
+        # GenericProvider
+        (
+            GenericProvider,
+            Path(dir_path, "data", "ical", "ical1"),
+            Path(dir_path, "data", "ical", "ical1_result.json"),
+        ),
+        # EUNetworks
+        (EUNetworks, Path(dir_path, "data", "ntt", "ntt1"), Path(dir_path, "data", "ntt", "ntt1_result.json"),),
```

Comment:
Why test `ntt` data with `EUNetworks` provider? Same question regarding PacketFabric and Telstra below.

Reply:
it's an error, it should use the generic ICal parser